.gitignore

```diff
@@ -9,4 +9,4 @@ sites/*/public
 .venv
 .DS_Store
 .idea/
-package-lock.json
\ No newline at end of file
+package-lock.jsonte
```

Comment:
Is this intended?

Reply:
Hmm, weird, my local file doesn't have it, but my Local History shows it was briefly there. No, it's not intended.

Reply:
```suggestion
package-lock.json
```